tracing-serde/Cargo.toml

```diff
@@ -24,7 +24,7 @@ tracing-core = "0.1.2"
 
 [dev-dependencies]
 serde_json = "1.0"
-tracing = "0.1.3"
+tracing = "0.1"
```

Comment:
This was broken when I was trying to build because it couldn't find `#[tracing::instrument]`, no idea if this is the right way to fix it.

Reply:
If you just ran `cargo update` it would also have fixed it. We might want to back this out since it's unrelated, but I don't really care.